javascript/javascript3/week1/styles.css

```diff
@@ -0,0 +1,108 @@
+#container{
+    background-image: url("https://www.wallpaperuse.com/wallp/25-252084_m.jpg");
+    height:500px;
+    width:400px;
+    border-radius:10px;
+    font-family: 'Noto Sans Display', sans-serif;
+    margin:auto;
+    
+  }
+  .top-button{
+    text-align: center;
+    margin-top:10px;
+  }
+  #location{
+      background-color: #004d99;
+      color:azure;
+      font-size: 18px;
+      padding:5px;
+      text-align: center;
+      border-radius: 10px;;
+  }
+  .top-container{
+    text-align:center;
+    
+  }
+  #heading{
+    padding-top:15px;
+    font-style:bold;
+    font-weight:600;
+    
+  }
+  #temppara{
+    font-style:bold;
+    font-weight:400;
+  }
+  #feelslike{
```

Comment:
Regular convention for class and id names would be using hyphens. Try implementing that :)